src/webgpu/shader/execution/expression/call/builtin/textureStore.spec.ts

```diff
@@ -121,6 +128,265 @@ Parameters:
   )
   .unimplemented();
 
+// Returns shader input values for texel format tests.
+// Values are intentionally simple to avoid rounding issues.
+function inputArray(format: string): number[] {
```

Comment:
Also worth mentioning that the snorm and unorm values go a bit outside the range, to demonstrate clamping behaviour.

(Heh. Just checked, and I don't think the spec describes that clamping. I added it to the reuquired-for-CR list in the speck writing plan)